puppet/modules/slave/manifests/docker.pp

```diff
@@ -19,11 +19,7 @@
     use_upstream_package_source => false,
     service_overrides_template  => false,
     docker_ce_package_name      => $docker_ce_package_name,
+    ensure                      => absent,
   }
 
-  group { 'docker':
-    ensure => 'present',
-  }
-
-  User<|title == 'jenkins'|>{groups +> ['docker']}
```

Comment:
I dropped this, but didn't make it remove jenkins from that group, as I didn't know if it won't explode trying to do so?

Reply:
I'd expect you can ensure the group to be absent as well.

Reply:
no risk, no fun